collector/analyzer/uprobeanalyzer/uprobeanalyzer.go

```diff
@@ -11,8 +12,11 @@ import (
 	"github.com/Kindling-project/kindling/collector/model/constvalues"
 	"github.com/hashicorp/go-multierror"
 	"go.uber.org/zap"
+	"strings"
 )
 
+var httpMerger = tools.NewHttpMergeCache()
```

Comment:
It seems like tools.httpMerger is used in two different packages now, then we should remove this tool from `networkanalyzer` and` put it into a new package called `analyzerhelper` under package `analyzer` to decouple networkanalyzer from uprobeanalyzer.